.gitattributes

```diff
@@ -2,7 +2,7 @@
 *.[ch] whitespace=indent,trail,space diff=cpp
 *.sh whitespace=indent,trail,space eol=lf
 *.perl eol=lf diff=perl
-*.pl eof=lf diff=perl
```

Comment:
No

Reply:
This is wrong.

Reply:
@e-kwsm can you please fix it? The `git rebase -i` command will come in handy.